### ISSUE TITLE
Schematics to structure project

### ISSUE THREAD

**Issue by valorkin** [valorkin/module-federation#33]
@skocharian commented on [Wed Sep 30 2020](https://github.com/valorkin/ng-mf-submodules/issues/21)

Create _generate_ schematics that will preconfigure webpack config. But this should have a dependency on current mf-builder work, if we get more closer to the ng commands - so this one probably has a lower priority than the other tasks. Low priority

Once MF: Resources & scss is done we need to automate this with set of schematics.

Create schematics that can generate Webpack configuration based on the angular JSON
-- New project: @fundamental-ngx/app-shell:ng-add
-- Existing project: @fundamental-ngx/app-shell:generate schematics
Reorg- existing angular structure to match our ng-mf-submodules

---

@fkolar commented on [Wed Oct 14 2020](https://github.com/valorkin/ng-mf-submodules/issues/21#issuecomment-708584887)

@skocharian let;s move this to M3, as this will have dependency to a task where we integrate CLI build with ngx-build-plus
